manager/src/main/resources/db/migration/0.1/V0.1.10__update_connector.sql

```diff
@@ -0,0 +1,15 @@
+-- rename "connector_type" to "connector_type_id"
+alter table CONNECTOR
+    rename column connector_type to connector_type_id;
+
+-- add new "type" column
+alter table CONNECTOR
+    add column "type" varchar(255);
+
+-- update existing connectors "type" with 'SINK' value (the only possible one before this migration)
+update CONNECTOR
+    set "type"='SINK';
```

Comment:
Would it not be safer to include `where type is null`?

Reply:
there can't be source connectors before merging this PR

Reply:
Indeed, but there can be afterwards and this is re-deployed to an environment unless we always delete the database.

It was only a comment; I approved anyway.